### PR TITLE
Changes for TimescaleDB 2.0.0-rc2 launch

### PR DESCRIPTION
**Pull request by ryanbooz**
**NOTE**: These changes should ONLY be applied to the 2.0 branch. A separate PR will target 1.7 branch with appropriate changes there.

These changes include:
* removed "Update Software" from the "Using TimescaleDB" section
* changes to the Install pages, including:
  * removing unsupported install types
  * modifying all supported install docs to point to 2.0.0-rc1 (and other verbiage as appropriate)
* addition of a new main navigation category "Update TimescaleDB"
* sub-pages below the new category for detailed install docs
* addition of a migration/changes doc for 2.0 under "Release Notes"

### CODE REVIEW

File: getting-started/installation-apt-debian.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
->:WARNING: Our scaling out capabilities are currently in BETA and
-are not available through this installation method. For more information,
-please [contact us][contact] or join the #multinode-beta channel in our 
+>:WARNING: These installation instructions will install the current Release Candidate
+for TimescaleDB 2.0, which includes the ability to setup [Multi-Node capabilities][multi-node-basic]. For
```
Comment (mfreed) [inline getting-started/installation-apt-debian.md:2]: Minor nit:  I think we've generally been keeping "multi-node" lower case, just for consistency.

File: getting-started/installation-apt-ubuntu.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
->:WARNING: Our scaling out capabilities are currently in BETA and
-are not available through this installation method. For more information,
-please [contact us][contact] or join the #multinode-beta channel in our 
+>:WARNING: These installation instructions will install the current Release Candidate
+for TimescaleDB 2.0, which includes the ability to setup [Multi-Node capabilities][multi-node-basic]. For
+more information, please [contact us][contact] or join the #multinode-beta channel in our
```
Comment (mfreed) [inline getting-started/installation-apt-ubuntu.md:3]:
```suggestion
more information, please [contact us][contact] or join the #multinode channel in our 
```

Comment (mfreed) [reply getting-started/installation-apt-ubuntu.md:3]: I renamed from #multinode-beta to #multinode today, please check various places this might be included?

File: getting-started/installation-docker.md [hunk 3]
```diff
@@ -5,7 +5,7 @@
 Start a TimescaleDB instance, pulling our Docker image from [Docker Hub][] if it has not been already installed:
 
 ```bash
-docker run -d --name timescaledb -p 5432:5432 -e POSTGRES_PASSWORD=password timescale/timescaledb:latest-pg:pg_version:
+docker run -d --name timescaledb -p 5432:5432 -e POSTGRES_PASSWORD=password timescale/timescaledb:x.y.z-pg:pg_version:
```
Comment (mfreed) [inline getting-started/installation-docker.md:8]: Do we want to have as `x.y.z`, or explicitly label it for now as `2.0.0`, and then switch back to `latest` when 2.0 is in GA?  The reason it was latest was so that people could just cut/paste.

File: release-notes.md [hunk 4]
```diff
@@ -15,11 +15,11 @@ can view active developments on GitHub at any time.
 The team is actively working on the multi-node version
 of TimescaleDB which is currently in beta. You can read more about our
 architecture and design for distributed hypertables
-[here](https://docs.timescale.com/beta-v2.0.0/introduction/architecture#distributed-hypertables).
+[here](https://docs.timescale.com/v2.0/introduction/architecture#distributed-hypertables).
```
Comment (mfreed) [inline release-notes.md:18]: Oops, I just submitted a PR for this section, these will conflict.  

https://github.com/timescale/docs.timescale.com-content/pull/544

One of us can handle the merge later.  I think your later TIP and link to Changes should remain.

File: update-timescaledb.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# Updating TimescaleDB versions [](update)
+
+This section describes how to upgrade between different versions of
+TimescaleDB. TimescaleDB supports **in-place updates**:
+you don't need to dump and restore your data, and versions are published with
+automated migration scripts that convert any internal state if necessary.
```
Comment (mfreed) [inline update-timescaledb.md:6]: Do we want to explicitly say we currently only support forward updates / upgrades, not backwards / downgrades?

File: update-timescaledb.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+### Upgrade TimescaleDB
+
+To upgrade an existing TimescaleDB instance, follow the documentation below based on
+your current upgrade path.
+
+**TimescaleDB 2.0**: [Updating TimescaleDB from 1.x to 2.0-RC1+][update-tsdb-2]
+
+**TimescaleDB 2.0 on Docker**: [Update TimescaleDB on Docker from 1.7.4 to 2.0-RC1+][update-docker]
+
+**TimescaleDB 1.x**: [Update TimescaleDB 1.x to 1.7.4][update-tsdb-1]
```
Comment (mfreed) [inline update-timescaledb.md:29]:
```suggestion
**TimescaleDB 1.x**: [Updating TimescaleDB 1.x to 1.7.4][update-tsdb-1]
```

File: update-timescaledb.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+>:TIP:If you need to upgrade PostgreSQL first, please see [our documentation][upgrade-pg].
+
+### Upgrade TimescaleDB
+
+To upgrade an existing TimescaleDB instance, follow the documentation below based on
+your current upgrade path.
+
+**TimescaleDB 2.0**: [Updating TimescaleDB from 1.x to 2.0-RC1+][update-tsdb-2]
+
+**TimescaleDB 2.0 on Docker**: [Update TimescaleDB on Docker from 1.7.4 to 2.0-RC1+][update-docker]
```
Comment (mfreed) [inline update-timescaledb.md:27]:
```suggestion
**TimescaleDB 2.0 on Docker**: [Updating TimescaleDB on Docker from 1.7.4 to 2.0-RC1+][update-docker]
```

File: update-timescaledb/update-docker.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
+# Updating a TimescaleDB Docker installation
+
+As a more concrete example, the following steps should be taken with a docker
```
Comment (mfreed) [inline update-timescaledb/update-docker.md:3]:
```suggestion
The following steps should be taken with a docker
```

File: update-timescaledb/update-tsdb-2.md [hunk 9]
```diff
@@ -0,0 +1,187 @@
+# Updating TimescaleDB to 2.0 [](update)
+
+Use these instructions to update TimescaleDB 1.x to TimescaleDB 2.0
+
+
+>:WARNING:These instructions are only for upgrading TimescaleDB 1.x to TimescaleDB 2.0
+> If you need to upgrade your existing TimescaleDB 1.x to a newer version in the 1.x
+> release line (eg. 1.7.2 to 1.7.4), please see Update [TimescaleDB 1.x][update-tsdb-1]
```
Comment (mfreed) [inline update-timescaledb/update-tsdb-2.md:8]:
```suggestion
> release line (eg. 1.7.2 to 1.7.4), please see Update [TimescaleDB 1.x][update-tsdb-1].
```

File: update-timescaledb/update-tsdb-2.md [hunk 10]
```diff
@@ -0,0 +1,187 @@
...
+ --------------------|-------------------------------
+ 2.0-RC1+            | 11, 12
+
+>:TIP:If you need to upgrade PostgreSQL first, please see [our documentation][upgrade-pg].
+
+### Notice of breaking changes from TimescaleDB 1.3+
+TimescaleDB 2.0 supports **in-place updates** just like previous releases. During the update, scripts will automatically configure
+updated features to work as expected with TimescaleDB 2.0.
+
+Because this is our first major version release in two years, however, we’re providing additional guidance to help you ensure the update completes successfully and everything is configured as expected (and optimized for your workload). In particular, settings related to [Continuous Aggregates][caggs], [compression][compression], and [data retention][retention] have been modified to provide greater configuration transparency and flexibility, therefore we highly recommend verifying that these settings were migrated correctly.
```
Comment (mfreed) [inline update-timescaledb/update-tsdb-2.md:25]: Small nit - generally recommend using hard line breaks, so that later diffs can be more easily compared via git at the line (vs. paragraph) level.

File: update-timescaledb/update-tsdb-2.md [hunk 11]
```diff
@@ -0,0 +1,187 @@
...
+
+**Before completing the upgrade**, we encourage you to read [Changes in TimescaleDB 2.0][changes-in-ts2] for a more detailed look at the major changes in TimescaleDB 2.0 and how they impact the way your applications and scripts interact with the API.
+
+### Prerequisites
```
Comment (mfreed) [inline update-timescaledb/update-tsdb-2.md:29]: And anchor tags so we can in-page link?

File: update-timescaledb/update-tsdb-2.md [hunk 12]
```diff
@@ -0,0 +1,187 @@
...
+For example, if you are currently running PostgreSQL 10 and TimescaleDB 1.5, the recommended upgrade path to PostgreSQL 12 and TimescaleDB 2.0 would be:
+
+1. Update TimescaleDB 1.5 to TimescaleDB 1.7 on PostgreSQL 10
+1. Upgrade PostgreSQL 10 to PostgreSQL 12 with TimescaleDB 1.7 installed
+1. Update TimescaleDB 1.7 to TimescaleDB 2.0 on PostgreSQL with the instructions below
+
+>:TIP: Whenever possible, prefer the most recent supported version, PostgreSQL 12. Please see our [Upgrading PostgreSQL][upgrade-pg] guide for help.
+
+#### Fix Continuous Aggregate Errors Before Upgrading
+Before starting the upgrade to TimescaleDB 2.0, **we highly recommend checking the database log for errors related to failed retention policies that were occurring in TimescaleDB 1.x** and then either remove them or update them to be compatible with existing continuous aggregates. Any remaining retention policies that are still incompatible with the 'ignore_invalidation_older_than' setting will automatically be disabled during the upgrade and a notice provided.
```
Comment (mfreed) [inline update-timescaledb/update-tsdb-2.md:42]: Backticks for code block formatting

```suggestion
Before starting the upgrade to TimescaleDB 2.0, **we highly recommend checking the database log for errors related to failed retention policies that were occurring in TimescaleDB 1.x** and then either remove them or update them to be compatible with existing continuous aggregates. Any remaining retention policies that are still incompatible with the `ignore_invalidation_older_than` setting will automatically be disabled during the upgrade and a notice provided.
```

File: update-timescaledb/update-tsdb-2.md [hunk 13]
```diff
@@ -0,0 +1,187 @@
...
+
+>:TIP:Read more about changes to continuous aggregates and data retension policies [here][retention-cagg-changes]
```
Comment (mfreed) [inline update-timescaledb/update-tsdb-2.md:44]:
```suggestion
>:TIP:Read more about changes to continuous aggregates and data retension policies [here][retention-cagg-changes].
```

**Review by mfreed — COMMENTED** [review]
Looks good to me (modulo some suggestions).  Leaving as "comment", however, for @solugebefola explicit approval given all the page-level changes.

File: getting-started/installation-docker.md (hunk 3 above)
Comment (ryanbooz) [reply getting-started/installation-docker.md:8]: turns out that `x.y.z` (locally at least) is replaced at runtime with the branch of docs you're looking at. So, this will end up being `timescale/timescaledb:2.0.0-rc1-pg12`

@solugebefola @tedsczelecki - am I correct on that?

File: release-notes.md (hunk 4 above)
Comment (ryanbooz) [reply release-notes.md:18]: np. I'll merge again afterwards and deal with it.

File: update-timescaledb/update-tsdb-2.md (hunk 9 above)
Comment (ryanbooz) [reply update-timescaledb/update-tsdb-2.md:8]: Oops! I meant to go back through and look at periods in all docs, especially bulleted lists.  Thanks!

**Review by mfreed — APPROVED** [review]

File: getting-started/release-notes/changes-in-timescaledb-2.md [hunk 14]
```diff
@@ -0,0 +1,290 @@
...
+```SQL
+CALL refresh_continuous_aggregate(
+  continuous_aggregate => 'conditions_by_2h',
+  window_start         => '2020-05-01', 
+  window_end           => '2020-05-03');
+```
+
+Users can use this command explicitly; a typical use of this command is to manually refresh historical data while creating a continuous aggregate policy to aggregate new data. It is even possible to build custom continuous aggregation policies using this function within the new user-defined action framework which are then scheduled via `add_job`. 
+
+Note that` refresh_continuous_aggregate` only recomputes the aggregated time buckets that completely fall inside the given refresh window boundaries and are in a region that has seen changes in the underlying hypertable. Thus, if no changes have occurred in the underlying source data (that is, no data has been backfilled to the region or no updates to existing data have been made), no materialization will be performed either over that region. This behavior is similar to the continuous aggregate policy and ensures more efficient operation.
```
Comment (solugebefola) [inline getting-started/release-notes/changes-in-timescaledb-2.md]:
```suggestion
Note that` refresh_continuous_aggregate` only recomputes the aggregated time buckets that completely fall inside the given refresh window and are in a region that has seen changes in the underlying hypertable. Thus, if no changes have occurred in the underlying source data (that is, no data has been backfilled to the region or no updates to existing data have been made), no materialization will be performed either over that region. This behavior is similar to the continuous aggregate policy and ensures more efficient operation.
```

File: getting-started/release-notes/changes-in-timescaledb-2.md [hunk 15]
```diff
@@ -0,0 +1,290 @@
...
+
+
+## Using data retention and continuous aggregates together [](retention-and-caggs)
+
+With greater power and flexibility also comes a greater responsibility to understand how features interact. Specifically, users should understand the interactions between data retention policy settings and continuous aggregate settings. 
+
+Before starting the upgrade to TimescaleDB 2.0, **we highly recommend checking the database log for errors related to failed retention policies that were occurring in TimescaleDB 1.x** and then either remove them or update them to be compatible with existing continuous aggregates. Any remaining retention policies that are still incompatible with the `ignore_invalidation_older_than `setting will automatically be disabled with a notice during the upgrade.
```
Comment (solugebefola) [inline getting-started/release-notes/changes-in-timescaledb-2.md]:
```suggestion
Before starting the upgrade to TimescaleDB 2.0, **we highly recommend checking the database log for errors related to failed retention policies that were occurring in TimescaleDB 1.x** and then either removing them or updating them to be compatible with existing continuous aggregates. Any remaining retention policies that are still incompatible with the `ignore_invalidation_older_than` setting will automatically be disabled with a notice during the upgrade.
```
To maintain consistency in tense.  TIWAGOS

File: getting-started/release-notes/changes-in-timescaledb-2.md [hunk 16]
```diff
@@ -0,0 +1,290 @@
...
+
+As an example, if a data retention policy on a hypertable is set for `drop_after => '4 weeks'`, then the policy associated with a continuous aggregate on that same hypertable should have a `start_offset` less than or equal to 4 weeks.  Similarly, any manual call to `refresh_continuous_aggregate` should likely specify a `window_start` that’s also less than the date from 4 weeks ago. 
+
+If not understood properly, users could overwrite existing aggregated data in continuous aggregates with empty data by recomputing data over a now-dropped time window of data, not likely the result a user was expecting. Instead, users seeking to keep only a certain time window of data in their continuous aggregate view should create a data retention policy on the continuous aggregate.
```
Comment (solugebefola) [inline getting-started/release-notes/changes-in-timescaledb-2.md]:
```suggestion
If not understood properly, users could overwrite existing aggregated data in continuous aggregates with empty data by recomputing data over a now-dropped time window of data, not likely the result a user was expecting. Instead, users seeking to keep only a certain time window of data in their continuous aggregate view should create a data retention policy on the continuous aggregate itself.
```

File: getting-started/release-notes/changes-in-timescaledb-2.md [hunk 17]
```diff
@@ -0,0 +1,290 @@
...
+
+After upgrading to TimescaleDB 2.0, **retention policies will no longer fail due to incompatibilities with continuous aggregates** and users have to ensure that retention and continuous aggregate policies have the desired interplay. 
+
+## Differences in the handling of backfill on continuous aggregates
+
+In TimescaleDB 1.x, data that was backfilled into hypertables wasn’t handled optimally; modification of any hypertable data, regardless of how old, would cause the continuous aggregate materializer job to restart from the point of the earliest backfill and then it would materialize from that point forward. Unfortunately, this could also cause the materializer to get “stuck”, since the background job only processed a limited amount of data per run, as governed by the `max_interval_per_job` setting. When this happened, one job run could potentially force the next job to start all over again.
```
Comment (solugebefola) [inline getting-started/release-notes/changes-in-timescaledb-2.md]:
```suggestion
In TimescaleDB 1.x, data that was backfilled into hypertables wasn’t handled optimally; modification of any hypertable data, regardless of how old, would cause the continuous aggregate materializer job to restart from the point of the earliest backfill and then materialize from that point forward. Unfortunately, this could also cause the materializer to get “stuck”, since the background job only processed a limited amount of data per run, as governed by the `max_interval_per_job` setting. When this happened, one job run could potentially force the next job to start all over again. 
```

File: getting-started/release-notes/changes-in-timescaledb-2.md [hunk 18]
```diff
@@ -0,0 +1,290 @@
...
+
+In contrast, TimescaleDB 2.0 never stops tracking backfill, and to avoid materializing too much historical data, one should simply use a refresh window that does not include that region of data. The backfilled region can always be refreshed at a later time, either manually or via a policy.
+
+To ensure that previously ignored backfill can be refreshed after the upgrade to TimescaleDB 2.0, the upgrade process will mark the region older than the `ignore_invalidation_older_than` threshold as “requiring refresh”. This allows a manual refresh to bring a continuous aggregate up-to-date with the underlying source data. If the `ignore_invalidation_older_than` threshold was reset to a larger interval at some point, we recommend setting it back to the smaller interval prior to upgrading to ensure that all the backfill can be refreshed, if one so desires.
```
Comment (solugebefola) [inline getting-started/release-notes/changes-in-timescaledb-2.md]: Not sure what 'larger interval' is referring to here?

File: getting-started/release-notes/changes-in-timescaledb-2.md [hunk 19]
```diff
@@ -0,0 +1,290 @@
...
+
+In contrast, TimescaleDB 2.0 never stops tracking backfill, and to avoid materializing too much historical data, one should simply use a refresh window that does not include that region of data. The backfilled region can always be refreshed at a later time, either manually or via a policy.
+
+To ensure that previously ignored backfill can be refreshed after the upgrade to TimescaleDB 2.0, the upgrade process will mark the region older than the `ignore_invalidation_older_than` threshold as “requiring refresh”. This allows a manual refresh to bring a continuous aggregate up-to-date with the underlying source data. If the `ignore_invalidation_older_than` threshold was reset to a larger interval at some point, we recommend setting it back to the smaller interval prior to upgrading to ensure that all the backfill can be refreshed, if one so desires. 
+
+Note again, however, that if backfill was previously ignored due to a retention policy on the underlying hypertable, a manual refresh of older data into a continuous aggregate could remove data when hypertable chunks have been dropped due to a data retention policy as discussed in the previous section.
```
Comment (solugebefola) [inline getting-started/release-notes/changes-in-timescaledb-2.md]: This is slightly hard to parse?  Maybe just:
> Note again, if older data has been dropped due to a data retention policy, manual refresh of data from that time range into the continuous aggregate could remove data from the aggregate .

Or something similar

File: getting-started/release-notes/changes-in-timescaledb-2.md [hunk 20]
```diff
@@ -0,0 +1,290 @@
...
+
+## Viewing information about continuous aggregates
+
+In TimescaleDB 2.0, views surrounding continuous aggregates (and other policies) have been simplified and generalized.
+
+### Changes and Additions
+*   [timescaledb_information.continuous_aggregates](https://docs.timescale.com/v2.0/api#timescaledb_information-continuous_aggregate): now provides information related to the materialized view, which includes the view name and owner, the real time aggregation flag, the materialization and the view definition (the select statement defining the view)
+*   [timescaledb_information.jobs](https://docs.timescale.com/v2.0/api#timescaledb_information-jobs): displays information for all policies including continuous aggregates  
+*   [Timescaledb_information.job_stats](https://docs.timescale.com/v2.0/api#timescaledb_information-jobs_stats): displays job statistics related to all jobs
```
Comment (solugebefola) [inline getting-started/release-notes/changes-in-timescaledb-2.md]:
```suggestion
*   [timescaledb_information.job_stats](https://docs.timescale.com/v2.0/api#timescaledb_information-jobs_stats): displays job statistics related to all jobs
```

File: getting-started/release-notes/changes-in-timescaledb-2.md [hunk 21]
```diff
@@ -0,0 +1,290 @@
...
+
+# Other Superficial API Changes
+
+Other minor changes were made to various APIs for greater understandability and consistency, including in the following areas.
+
+## Data Retention
+
+### Changes and Additions
+*   [drop_chunks](https://docs.timescale.com/v2.0/api#drop_chunks): This function now requires specifying a hypertable or continuous aggregate as the first argument, and does not allow dropping chunks across all hypertables in a database.  Additionally, the arguments `cascade` and `cascade_to_materializations` were removed (and behave as if the arguments were set to `false` in earlier versions). In TimescaleDB 2.0, we instead recommend creating a separate retention policy on each continuous aggregate. 
+*   [add_retention_policy](https://docs.timescale.com/v2.0/api#add_retention_policy), [remove_retention_policy](https://docs.timescale.com/v2.0/api#remove_retention_policy):  Creating (or removing) a data retention policy now have explicit functions. Additionally, the arguments `cascade` and `cascade_to_materializations` were removed (and behave as if the arguments were set to `false` in earlier versions).
```
Comment (solugebefola) [inline getting-started/release-notes/changes-in-timescaledb-2.md]:
```suggestion
*   [add_retention_policy](https://docs.timescale.com/v2.0/api#add_retention_policy), [remove_retention_policy](https://docs.timescale.com/v2.0/api#remove_retention_policy):  Creating (or removing) a data retention policy now has explicit functions. Additionally, the arguments `cascade` and `cascade_to_materializations` were removed (and behave as if the arguments were set to `false` in earlier versions).
```

File: getting-started/release-notes/changes-in-timescaledb-2.md [hunk 22]
```diff
@@ -0,0 +1,290 @@
...
+*   [timescaledb_information.jobs](https://docs.timescale.com/v2.0/api#jobs): General information about data retention policies are now available in the main jobs view
```
Comment (solugebefola) [inline getting-started/release-notes/changes-in-timescaledb-2.md]:
```suggestion
*   [timescaledb_information.jobs](https://docs.timescale.com/v2.0/api#jobs): General information about data retention policies are now available in the main jobs view.
```

File: getting-started/release-notes/changes-in-timescaledb-2.md [hunk 23]
```diff
@@ -0,0 +1,290 @@
...
+
+### Removed
+*   [add_drop_chunks_policy](https://docs.timescale.com/v1.7/api#add_drop_chunks_policy): removed in favor of the explicit functions above
+*   [timescaledb_information.drop_chunks_policies](https://docs.timescale.com/v1.7/api#timescaledb_information-drop_chunks_policies) view has been removed in favor of the more general jobs view.
+
+
+## Compression
+
+### Changes and Additions
+*   [add_compression_policy](https://docs.timescale.com/v2.0/api#add_compression_policy), [remove_compression_policy](https://docs.timescale.com/v2.0/api#remove_compression_policy):  Creating (or removing) a compression policy now have explicit functions
```
Comment (solugebefola) [inline getting-started/release-notes/changes-in-timescaledb-2.md]:
```suggestion
*   [add_compression_policy](https://docs.timescale.com/v2.0/api#add_compression_policy), [remove_compression_policy](https://docs.timescale.com/v2.0/api#remove_compression_policy):  Creating (or removing) a compression policy now has explicit functions
```

File: getting-started/release-notes/changes-in-timescaledb-2.md [hunk 24]
```diff
@@ -0,0 +1,290 @@
...
+*   [timescaledb_information.jobs](https://docs.timescale.com/v2.0/api#jobs): General information about data retention policies are now available in the main jobs view
+
+### Removed
+*   [add_drop_chunks_policy](https://docs.timescale.com/v1.7/api#add_drop_chunks_policy): removed in favor of the explicit functions above
+*   [timescaledb_information.drop_chunks_policies](https://docs.timescale.com/v1.7/api#timescaledb_information-drop_chunks_policies) view has been removed in favor of the more general jobs view.
+
+
+## Compression
+
+### Changes and Additions
```
Comment (solugebefola) [inline getting-started/release-notes/changes-in-timescaledb-2.md:418]: Some inconsistency about whether list items have periods on the end

File: getting-started/release-notes/changes-in-timescaledb-2.md [hunk 25]
```diff
@@ -0,0 +1,290 @@
...
+* [timescaledb_information.compressed_hypertable_stats](https://docs.timescale.com/v1.7/api#timescaledb_information-compressed_hypertable_stats): removed in favor of the new `hypertable_compression_stats(hypertable)` function linked above
+* [timescaledb_information.compressed_chunk_stats](https://docs.timescale.com/v1.7/api#timescaledb_information-compressed_chunk_stats): removed in favor of the new `chunk_compression_stats(hypertable)` function linked above
+
+## Managing policies and other jobs
+
+TimescaleDB 2.0 introduces user-defined actions and creates a more unified jobs API. Now, jobs created by the TimescaleDB policies and for user-defined actions can be managed and viewed through a single API.
+
+
+
+*   `add_job:` Adds a new user-defined action to the job scheduling framework.
```
Comment (solugebefola) [inline getting-started/release-notes/changes-in-timescaledb-2.md:443]: No links? Are these functions not in the API yet?

**Review by solugebefola — COMMENTED** [review]
LG in general!  A few things, some TIWAGOS.  Of note for the whole doc:
1) GitHub doesn't do line-wrapping, so paragraphs without line breaks make it hard to comment on individual lines.  If you can use whatever IDE you use to add line breaks at ~80 characters, that would make it easier to review.
2) Some inconsistency w.r.t. list items having periods at the end or not
3) Please change all links to footnote-style links rather than inline

File: getting-started/release-notes/changes-in-timescaledb-2.md [hunk 26]
```diff
@@ -0,0 +1,290 @@
...
+
+To facilitate many of the improvements in [TimescaleDB 2.0](https://github.com/timescale/timescaledb/releases/tag/2.0.0-rc1), several existing APIs and function definitions have been modified which may require updates to your existing code. 
+
+Most notably, the following features include updates and changes that may impact existing code using these interfaces. If your workloads use any of the features, this document covers each in detail to help you understand what - if any - action you need to take.
+
+*   **Dropping support for PostgreSQL versions 9.6 and 10:** As mentioned in [our upgrade documentation](https://docs.timescale.com/update-timescaledb), TimescaleDB 2.0 no longer supports older PostgreSQL versions. You will need to be running PostgreSQL version 11 or 12 to upgrade your TimescaleDB installation to 2.0.
```
[the remark this answers is not in the document]
Comment (ryanbooz) [reply getting-started/release-notes/changes-in-timescaledb-2.md]: Reworded slightly

File: getting-started/release-notes/changes-in-timescaledb-2.md [hunk 27]
```diff
@@ -0,0 +1,290 @@
...
+
+The following migration document provides more information on each of these API changes and how they affect users of TimescaleDB 1.x . We’ve also included a bit of information about the impetus for these changes and our design decisions. For a more in-depth coverage of APIs, as well as a detailed description of new capabilities included in 2.0, such as distributed hypertables and the ability to run user-defined actions (custom background jobs), please [review our updated documentation](https://docs.timescale.com/v2.0/main) (navigate to specific features of interest).
+
+Once you have read through this guide and understand the impact upgrading to the latest version may have on your existing application and infrastructure, please follow our [upgrade to TimescaleDB 2.0](https://docs.timescale.com/latest/using-timescaledb/update-timescale/update-db-2) documentation. You will find straight-forward instructions and recommendations to ensure everything is updated and works correctly.
+
+
+# Why we’ve made these changes
```
[the remark this answers is not in the document]
Comment (ryanbooz) [reply getting-started/release-notes/changes-in-timescaledb-2.md]: OK. I tried to look at other Docs examples to see what we've done previously. Maybe a long-term review at some point to bring consistency.

File: getting-started/release-notes/changes-in-timescaledb-2.md [hunk 28]
```diff
@@ -0,0 +1,290 @@
...
+
+It’s been two years since we released TimescaleDB 1.0 in October 2018 with the ambition to make it easy to scale PostgreSQL for time-series workloads. Since that initial release, we’ve added many new capabilities: support for compression, continuous aggregates, data lifecycle management, and numerous optimizations, as well as two new underlying PostgreSQL versions (11 and 12). At the same time, the core of TimescaleDB hasn’t changed significantly: we still have hypertables, auto-partitioning, query optimizations, and a basic user experience that many have come to appreciate because things “just work”.
```
[the remark this answers is not in the document]
Comment (ryanbooz) [reply getting-started/release-notes/changes-in-timescaledb-2.md]: Going to leave ATM as this content came from the group-reviewed Doc.

File: getting-started/release-notes/changes-in-timescaledb-2.md (hunk 18 above)
Comment (ryanbooz) [reply getting-started/release-notes/changes-in-timescaledb-2.md]: See what you think of this minor change.
